packages/google_maps_flutter/google_maps_flutter_android/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 2.1.10
+
+* Splits from `google_maps_flutter` as a federated implementation.
```

Comment:
`Splits` -> `Split` or `Splits` -> `Splits Android implementation`

Reply:
The first violates our CHANGELOG style guide; I'll update to the second.